core/server/common/src/main/java/alluxio/cli/ValidateEnv.java

```diff
@@ -103,7 +113,9 @@
         PROXY_WEB_VALIDATION_TASK,
         MASTERS_SSH_VALIDATION_TASK,
         WORKERS_SSH_VALIDATION_TASK,
-        UFS_ROOT_VALIDATION_TASK
+        UFS_ROOT_VALIDATION_TASK,
+        ULIMIT_NUMBER_OF_OPEN_FILES_VALIDATION_TASK,
```

Comment:
I will migrate the common section from PR https://github.com/Alluxio/alluxio/pull/5865/files and put ulimit tasks there.